phpBB/phpbb/profilefields/manager.php

```diff
@@ -343,7 +343,7 @@ public function generate_profile_fields_template_headlines($restrict_option = ''
 		 * @var array	profile_cache	A copy of the profile cache to make additional checks
 		 * @since 3.1.6-RC1
 		 */
-		$vars = ['restrict_option', 'tpl_fields', 'profile_cache'];
+		$vars = array('restrict_option', 'tpl_fields', 'profile_cache');
```

Comment:
Short array syntax pls

Reply:
Checks are erring with array missing for $var on event.